### PR TITLE
8.0: Migration of account_tax_analysis

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

**Comment by tafaRU** [comment]
:+1:  (tested too)

File: account_tax_analysis/README.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+you are able to group accounting entries by Taxes (VAT codes)
+and/or financial accounts.
+
+This way you will find easily differences you may see between
+the OpenERP tax report and what you see in your books.
+
+Contributors
+============
+
+ * Vincent Renaville (Camptocamp SA)
```
Comment (pedrobaeza) [inline account_tax_analysis/README.rst:16]: @guewen have you made the migration? You should be here.

File: account_tax_analysis/account_tax_analysis.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        relation='account_tax_period_rel',
+        column1='tax_analysis',
+        column2='period_id',
+        string='Periods',
+        required=True,
+    )
+
+    @api.multi
+    def show_vat(self):
+        if not self.period_list:
```
Comment (pedrobaeza) [inline account_tax_analysis/account_tax_analysis.py]: Maybe it's better to put all periods in fiscal year when you don't select any period (like other financial reports )

**Comment by pedrobaeza** [comment]
You should also update pot file with the new string (Taxes analysis)

File: account_tax_analysis/account_tax_analysis.py (hunk 2 above)
Comment (guewen) [inline account_tax_analysis/account_tax_analysis.py]: I didn't want to change the features, but why not, I will change that

**Comment by guewen** [comment]
pot file updated

**Comment by sbidoul** [comment]
@guewen there is a couple of unused imports.

**Comment by guewen** [comment]
removed the unused imports

**Comment by sbidoul** [comment]
:+1:

**Pull request merged**